airflow/operators/check_operator.py

```diff
@@ -123,11 +120,10 @@ class ValueCheckOperator(BaseOperator):
     __mapper_args__ = {
         'polymorphic_identity': 'ValueCheckOperator'
     }
-    template_fields = ('sql', 'pass_value',)  # type: Iterable[str]
+    template_fields = ('sql', 'pass_value',)  # type: Tuple[str, ...]
```

Comment:
Isn't this type the same? (Or this a superset of the previous type)

Reply:
mypy is going crazy in many places now.  It was one of the attempts to do it.